src/components/RequestInfo/index.tsx

```diff
@@ -1,49 +1,58 @@
 import { Col, Row } from "antd";
+import { useTranslation } from "react-i18next";
 
-import type { Request } from "@/types/request";
+import type { Maybe, Request_Logs } from "@/graphql/generated";
+import formatTime from "@/utils/helpers/formatTime";
 
 import styles from "./index.module.less";
 
 import type { FC } from "react";
 
-const RequestInfo: FC<Request> = ({
+const RequestInfo: FC<Partial<Request_Logs> & { queryKey?: Maybe<string> }> = ({
   id,
   path,
   duration,
-  startTime,
-  endtTime,
+  start_time,
```

Comment:
in components all props should be in camelCase. 

In other way you can provide full object: `request` with type `Request`

type Request {
 id
 path
 duration
 start_time
 end_time
 query_key
}